reposilite-site/data/guides/features/dashboard.md

```diff
@@ -3,36 +3,33 @@ id: dashboard
 title: Dashboard
 ---
 
-Reposilite exposes frontend that is mounted to `/#/` as SPA application in Vue 3.
-You can disable frontend in [shared configuration](/guide/settings#shared-configuration)
-if you just want to disable an easy way to query your repository or to provide custom frontend implementation using [static files](/guide/static-files).
+Reposilite exposes a frontend that is mounted to `/#/` as [SPA (_single-page application_)](https://en.wikipedia.org/wiki/Single-page_application) in [Vue 3](https://vuejs.org/). You can disable the frontend in [shared configuration](/guide/settings#shared-configuration) if you want to disable an easy way to query your repository or to provide a custom frontend implementation using [static files](/guide/static-files).
 
 #### Browser
-Browser displays content of currently selected directory, respectively to provided authentication credentials. For not logged user it lists only public files, for authenticated one it also displays files that user has access to.
+Browser displays the content of the currently selected directory in respect to the provided authentication credentials. For users not logged in it lists only public files, for authenticated users it also displays files that the user has access to.
 
 ![Dashboard Browser Preview](/images/guides/dashboard-browser-preview.png)
 
-You can control the way Reposilite displays directory content by opening adjustments view through the icon in right top corner:
+You can control the way how Reposilite displays directory content by opening the adjustments view via the icon in the top-right corner:
```

Comment:
```suggestion
You can control how Reposilite displays directory content by opening the adjustments view via the icon in the top-right corner:
```

I'm not sure `the way how` is the best way of wording this, maybe just `how` would work better?

Reply:
You're right: https://ell.stackexchange.com/q/246368/80364. I let me confuse myself.